setup-sshd

```diff
@@ -51,7 +51,7 @@ if [[ ${JENKINS_SLAVE_SSH_PUBKEY} == ssh-* ]]; then
 fi
 
 # ensure variables passed to docker container are also exposed to ssh sessions
-env | grep _ >> /etc/environment
+# env | grep _ >> /etc/environment
```

Comment:
this is a breaking change, will people be relying on this do we think? 

cc @MarkEWaite @slide @oleg-nenashev

Reply:
In fact, it should not break anything. 

The [base image](https://github.com/docker-library/openjdk/blob/master/8/jdk/buster/Dockerfile) only exports `JAVA_HOME` that could be matched and our Dockerfile is only exports `JENKINS_AGENT_HOME` that should match, but instead Jenkins user should and will use its own `HOME` variable, since `JENKINS_AGENT_HOME` is a non-standard environment variable. All other variables aren't matched previously.

Reply:
```suggestion
```